marlowe/test/Spec/Marlowe/Semantics/Functions.hs

```diff
@@ -882,11 +872,11 @@ checkGetContinuation =
            case' <- elements [Case action contract, MerkleizedCase action contractHash]
            pure (input, case', contract, contractHash == contractHash')
     forAll' gen $ \(input, case', contract, hashesMatch) ->
-      _ALLOW_FAILED_CONTINUATION_ || case (input, case', getContinuation input case') of
-        (NormalInput{}    , Case{}          , contract') -> Just contract == contract'
-        (MerkleizedInput{}, MerkleizedCase{}, contract') -> (Just contract == contract') == hashesMatch
-        (_                , _               , Nothing  ) -> True
-        _                                                -> False
+      case (input, case', getContinuation input case') of
+        (NormalInput{}    , Case{}                                       , contract') -> Just contract == contract'
+        (MerkleizedInput _ contractHash _, MerkleizedCase _ contractHash', _        ) -> (contractHash == contractHash') == hashesMatch
```

Comment:
Wait, isn't this shortcutting the test? Not part of the PR but just noticed
```suggestion
        (MerkleizedInput _ contractHash _, MerkleizedCase _ contractHash', _        ) -> contractHash == contractHash'
```

Reply:
Thanks for catching this! The combination of the somewhat convoluted generator and my trying to make the pattern matching too succinct end in making one of the matches tautological. This is fixed in 51fafdc82035fb096495f534907ea823171cc3e5, which is verbose and straightforward.